pkg/kv/kvserver/client_raft_test.go

```diff
@@ -1319,7 +1319,7 @@ func TestRequestsOnFollowerWithNonLiveLeaseholder(t *testing.T) {
 
 	st := cluster.MakeTestingClusterSettings()
 	// This test is specifically designed for epoch based leases.
-	kvserver.OverrideLeaderLeaseMetamorphism(ctx, &st.SV)
+	kvserver.OverrideDefaultLeaseType(ctx, &st.SV, roachpb.LeaseEpoch)
```

Comment:
Is this a no-op? The `OverrideLeaderLeaseMetamorphism` does the same invocation.

Reply:
Not any more -- see the commit linked in the PR description.

Reply:
Oh I see, not since 3904daa7.